export_terraform_variables.sh

```diff
@@ -0,0 +1,9 @@
+#!\bin\sh
```

Comment:
I think the shebang should look like `#!/bin/sh`

Reply:
It looks like this file only contains environment variables for terraform. ICYMI you could put these in a `tfvars` file. https://www.terraform.io/docs/configuration/variables.html#variable-definitions-tfvars-files

I think I would probably go for that approach since it doesn't modify the environment by exporting variables. Up to you though 😄

Reply:
Definitely missed this, thanks!